<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProntoSoccorso</class>
 <widget class="QWidget" name="ProntoSoccorso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(230, 235, 239);</string>
  </property>
  <widget class="QLabel" name="InformazioniProfilo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>581</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
color: rgb(230, 235, 239);</string>
   </property>
   <property name="text">
    <string>Registra paziente</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Nome">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(0, 57, 97,0);
	color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}</string>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Nome</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Cognome">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Cognome</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_CodiceFiscale">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
}

QLineEdit:focus {
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}
                 

</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Codice Fiscale</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Cellulare">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Cellulare</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAnnulla">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
}
</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRegistra">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(0, 57, 97,0);
	color: rgb(230, 235, 239);
color: rgb(230, 235, 239);
border-radius: 10px;
}
QPushButton:hover {
background-color: rgba(40, 40, 40,0.5);
    }</string>
   </property>
   <property name="text">
    <string>Registra</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit {
setPlaceholderText:(&quot;Inserisci qui il testo...&quot;);
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid black;
}
QDateEdit::up-button{
        width: 0px;
}
 QDateEdit::down-button {
width: 0px;
}
QDateEdit:focus{
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat SemiBold</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(230, 235, 239,0.6);
background-color: rgba(0, 57, 97,0);</string>
   </property>
   <property name="text">
    <string>Dati personali del paziente</string>
   </property>
  </widget>
  <widget class="QLabel" name="Sheda">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>290</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);
border: 1px solid rgba(230, 235, 239,0.6);
border-radius: 5px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid rgba(230, 235, 239,0.6);
border-radius: 6px;

}

QComboBox::drop-down {
image: url(:/newPrefix/UI UFFICIALI/MATERIALE/freccia.png);
width: 20px; 
height: 20px;
padding: 5px;
}

QComboBox QAbstractItemView {
/* Personalizza il menu a discesa */
        /* Definire le altre proprietà di stile per il menu a discesa qui */
background-color: rgba(40, 40, 40,0.5);
color: rgb(230, 235, 239);
border-radius: 6px;
        
    }
</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Uomo</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Uomo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donna</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Altro...</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);

border-radius: 6px;
   

</string>
   </property>
   <property name="text">
    <string>Data di nascita :</string>
   </property>
  </widget>
  <widget class="QLabel" name="LogoPazienti">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>135</y>
     <width>160</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(230, 235, 239,0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../MATERIALE/IconaPazientiom.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_CittaDiNascita">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Città di nascita</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_Residenza">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Medium</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
background-color: rgba(0, 57, 97,0);
color: rgb(230, 235, 239);
border: 0px solid black;
border-radius: 6px;

}
                            
QLineEdit:focus {
border: 1px solid rgba(230, 235, 239,0.6);
background-color: rgba(40, 40, 40,0.5);
}</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Residenza</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../MATERIALE/Back.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>Back</zorder>
  <zorder>Sheda</zorder>
  <zorder>InformazioniProfilo</zorder>
  <zorder>LineEdit_Nome</zorder>
  <zorder>LineEdit_CodiceFiscale</zorder>
  <zorder>LineEdit_Cognome</zorder>
  <zorder>LineEdit_Cellulare</zorder>
  <zorder>ButtonAnnulla</zorder>
  <zorder>ButtonRegistra</zorder>
  <zorder>dateEdit</zorder>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
  <zorder>label_3</zorder>
  <zorder>LineEdit_CittaDiNascita</zorder>
  <zorder>LineEdit_Residenza</zorder>
  <zorder>LogoPazienti</zorder>
 </widget>
 <resources>
  <include location="../../../../../../asa.qrc"/>
 </resources>
 <connections/>
</ui>
